docs/source/web_api.rst

```diff
@@ -122,7 +122,7 @@ On success, the social card service produces an HTTP 200 status response contain
             </p>
             </div>
     </blockquote>
-    <script async src="http://mementoembed.ws-dl.cs.odu.edu/static/js/mementoembed.js" charset="utf-8"></script>
+    <script async src="http://mementoembed.ws-dl.cs.odu.edu/static/js/mementoembed-v20180806.js" charset="utf-8"></script>
```

Comment:
@shawnmjones if you are using version number in the file name just to avoid any browser caches, then you can use the number as query parameter instead of the file name (i.e., `mementoembed.js?v=20180806` instead of `mementoembed-v20180806.js`). This way, the server will ignore any query parameters when serving a static file and serve whatever version of the file is present with that name, but the browser would be fooled to burst the cache. The reason of this change suggestion is because if you are tracking that file and not generating as the build process then renaming the file on every release would be messy.